snmp/datadog_checks/snmp/config.py

```diff
@@ -180,25 +176,33 @@ def parse_metrics(self, metrics, enforce_constraints, warning, log):
         raw_oids = []
         table_oids = {}
         mibs_to_load = set()
+
+        def get_table_symbols(mib, table):
+            key = (mib, table)
+            if key in table_oids:
+                return table_oids[key][1]
```

Comment:
```suggestion
                table_object, symbols = table_oids[key]
                return symbols
```

or 

```suggestion
                _, symbols = table_oids[key]
                return symbols
```

for readability, not fan of `[1]`